app/app.py

```diff
@@ -34,6 +34,13 @@ def call_history(interviewer):
     return jsonify(get_call_history_records(interviewer, start_date, end_date, survey_tla))
 
 
+@app.route("/api/reports/call-history/<interviewer>/instruments")
```

Comment:
We talked about whether `instruments` was the correct language, it's `questionnaire` in the datastore.